Frontend/ui-driver/src/Screens/RegistrationScreen/View.purs

```diff
@@ -15,82 +15,198 @@
+  in
+    -- lottieLoaderView state push
+    Anim.screenAnimation
+      $ relativeLayout
+          [ height MATCH_PARENT
+          , width MATCH_PARENT
+          , orientation VERTICAL
+          , background Color.white900
+          , clickable true
+          , onBackPressed push (const BackPressed)
+          , afterRender
+              ( \action -> do
+                  _ <- push action
+                  pure unit
+              )
+              $ const (AfterRender)
+          ]
+      $ [ linearLayout
+            [ height MATCH_PARENT
+            , width MATCH_PARENT
+            , gravity CENTER
+            , orientation VERTICAL
+            ]
+            [ linearLayout
+                [ height WRAP_CONTENT
+                , width MATCH_PARENT
+                -- , visibility if state.props.logoutModalView then GONE else VISIBLE
+                ]
+                [ PrestoAnim.animationSet
+                    [ Anim.fadeIn true
+                    ]
+                    $ StepsHeaderModel.view (push <<< StepsHeaderModelAC) (stepsHeaderModelConfig state)
+                ]
+            , linearLayout
+                [ height WRAP_CONTENT
+                , width MATCH_PARENT
+                , orientation VERTICAL
+                , padding (Padding 16 16 16 0)
+                , visibility VISIBLE
+                , onClick push (const BackPressed)
+                , weight 1.0
+                ]
+                [ imageView
+                    [ width (V 20)
+                    , height (V 20)
+                    , imageWithFallback $ fetchImage FF_ASSET "ny_ic_back"
+                    , visibility GONE
+                    ]
+                , linearLayout
+                    [ width MATCH_PARENT
+                    , height WRAP_CONTENT
+                    , orientation VERTICAL
+                    ]
+                    [ linearLayout
+                        [ width MATCH_PARENT
+                        , height WRAP_CONTENT
+                        , margin $ MarginBottom 10
+                        ]
+                        [ textView
+                            $ [ width WRAP_CONTENT
+                              , height WRAP_CONTENT
+                              , text $ getVarString START_EARNING_IN_FOUR_STEPS [ show $ length state.data.registerationSteps - if (getValueToLocalNativeStore SHOW_SUBSCRIPTIONS /= "true") then 1 else 0 ]
+                              , weight 1.0
+                              ]
+                            <> FontStyle.body2 TypoGraphy
+                        , textView
+                            $ [ width WRAP_CONTENT
+                              , height WRAP_CONTENT
+                              , text $ show progressPercent <> "% " <> getString COMPLETE
+                              ]
+                            <> FontStyle.body2 TypoGraphy
+                        ]
+                    , linearLayout
```

Comment:
Better split the view in different different functions, instead for writing it in one function